fast64_internal/oot/oot_operators.py

```diff
@@ -130,13 +130,35 @@ def execute(self, context):
 			while nextIndex in indices:
 				nextIndex += 1
 			roomObj.ootRoomHeader.roomIndex = nextIndex
-			parentObject(bpy.context.scene.ootSceneExportObj, roomObj)
+			parentObject(sceneObj, roomObj)
 		
 		bpy.ops.object.select_all(action = "DESELECT")
 		roomObj.select_set(True)
 		context.view_layer.objects.active = roomObj
 		return {"FINISHED"}
 
+class OOT_AddCutscene(bpy.types.Operator):
+	# set bl_ properties
```

Comment:
```suggestion
```

Reply:
I included this as it appeared in other Operator classes in fast64 and it seemed to be the project's standard practice. Removed.